vagrant/boxes.d/99-local.yaml.example

```diff
@@ -59,3 +59,12 @@ centos7-dynflow-devel:
     variables:
       dynflow_devel_github_username: '<REPLACE ME>'
       dynflow_devel_github_fork_remote_name: 'origin'
+
+google-centos7-devel:
+  box: google-centos7 # always use this box_name for google
+  image_family: 'centos-7'
```

Comment:
Rather than specifying an image, Google uses the concept of image families to ensure users are using the most up to date image in an image family.